resources/2023-11-application-diagnostics-prototype.md

```diff
@@ -0,0 +1,264 @@
+# Title
+
+* **Status**: Pending
+* **Author**: `Ryan Nowak` (`@rynowak`)
+
+## Overview
+
+*This design-note describes a prototype we plan to build for application diagnostics. As this document is describing a prototype, it will be less complete than other design proposals.*
+
+Our goal is to prototype an end-to-end scenario for **application** diagnostics that emcompasses:
+
+- A platform engineer configures a metrics dashboard (Prometheus+Grafana) and a logging dashboard (probably Azure to start) and wiring these up to a Radius environment.
+- A developer deploys an application into the environment, and has metrics and log collection automatically configured for the application.
+- An SRE can configure alerts for the metrics. When they receive an alert, they can use the metadata from the metrics to:
+  - Navigate to the application graph and understand the application architecture.
+  - Navigate to the logs and craft a query to understand the problem.
```

Comment:
This is outside of Radius, right?

Reply:
The query part is outside of Radius. I think we could help with the navigation.